consumer/core/src/main/java/org/arquillian/pact/consumer/core/AbstractConsumerPactTest.java

```diff
@@ -19,11 +20,17 @@
 import org.jboss.arquillian.test.spi.event.suite.Test;
 
 import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.List;
 import java.util.Optional;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
 
 
 public abstract class AbstractConsumerPactTest {
 
+    private static final Logger logger = Logger.getLogger(AbstractConsumerPactTest.class.getName());
```

Comment:
My only point here is - this kind of discussion shouldn't happen in the code review at all, but if we finally add checkstyle and all the other tools to the mix we should make sure we follow some conventions and make exceptions if needed. For me it's fine having exception for `logger` but we should reflect it in code analysis tools.